.github/workflows/abtesting.yml

```diff
@@ -91,6 +90,35 @@ jobs:
         LEGACY: true
       run: ([ -z $plist_secret ] || scripts/third_party/travis/retry.sh scripts/test_quickstart.sh ABTesting true)
 
+  quickstart-ftl:
+    # Don't run on private repo unless it is a PR.
+    if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
```

Comment:
I believe this test should be 
```
 if: github.repository == 'Firebase/firebase-ios-sdk' 
```
since I don't think it will work from a forked repo since secrets won't be available

Reply:
Updated the comment to `# Don't run on private repo.`
I will remove the `pull_request` trigger later. Currently, I keep it to show the test result.
Another question, should I change the job name to `quickstart-ftl-cron-only`?

Reply:
Yes, making that name change would help with clarity.

Please remove the `pull_request` trigger before merging.